pandas/tests/frame/test_to_csv.py

```diff
@@ -1131,3 +1102,83 @@ def test_to_csv_with_dst_transitions(self):
             df.to_pickle(path)
             result = pd.read_pickle(path)
             assert_frame_equal(result, df)
+
+    def test_to_csv_quoting(self):
+        df = DataFrame({
+            'c_string': ['a', 'b,c'],
+            'c_int': [42, np.nan],
+            'c_float': [1.0, 3.2],
+            'c_bool': [True, False],
+        })
+
+        expected = """\
```

Comment:
Odd indentation is the MO it seems for anything CSV-related.  I'll leave for now unless there are really strong objections.  But thanks for mentioning though @TomAugspurger !